functions-and-operators/string-functions.md

```diff
@@ -121,13 +121,92 @@ Return number of characters in argument.
 
 Synonym for `CHAR_LENGTH()`.
 
-### [`CONCAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat)
+### [`CONCAT(str1,str2,...)`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat)
 
-Return concatenated string.
+The `CONCAT()` function is used to concatenate one or more arguments.
 
-### [`CONCAT_WS()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat-ws)
+```sql
+SELECT CONCAT('TiDB', NULL, 'Server');
++--------------------------------+
+| CONCAT('TiDB', NULL, 'Server') |
++--------------------------------+
+|                           NULL |
++--------------------------------+
+```
+
+`CONCAT()` returns `NULL` if any argument is `NULL`.
+
+```sql
+SELECT CONCAT('TiDB', ' ', 'Server', '-', 1, TRUE);
```

Comment:
I think we should split the SQL statement from the output like this:

````
```sql
SELECT CONCAT('TiDB', ' ', 'Server', '-', 1, TRUE);
```

```
+---------------------------------------------+
| CONCAT('TiDB', ' ', 'Server', '-', 1, TRUE) |
+---------------------------------------------+
|                              TiDB Server-11 |
+---------------------------------------------+
```
````